src/hooks/__internal__/useScrollBlock/scroll-block-manager.server.test.ts

```diff
@@ -0,0 +1,52 @@
+import ScrollBlockManager from "./scroll-block-manager";
+
+describe("ScrollBlockManager's registerComponent method behavior", () => {
+  const scrollBlockManager = new ScrollBlockManager();
+  const id1 = "id1";
+  scrollBlockManager.registerComponent(id1);
+
+  it("should set isBlocked to true after registering at least one component", () => {
+    expect(scrollBlockManager.isBlocked()).toBe(true);
+  });
+});
+
+describe("ScrollBlockManager behavior with registerComponent and unregisterComponent", () => {
+  it("isBlocked should still return true after unregisterComponent is called with only one of the registered IDs ('id1')", () => {
```

Comment:
comment(non-blocking): for consistency we have tried to ensure describe/it tests are written with continuous prose

Again, we will likely just pick this up once it's in and fix for consistency